pkg/cloud/services/gc/loadbalancer.go

```diff
@@ -128,3 +131,138 @@ func (s *Service) deleteTargetGroup(ctx context.Context, targetGroupARN string)
 
 	return nil
 }
+
+// describeLoadBalancers gets all elastic LBs.
+func (s *Service) describeLoadBalancers(ctx context.Context) ([]string, error) {
+	var names []string
+	err := s.elbClient.DescribeLoadBalancersPagesWithContext(ctx, &elb.DescribeLoadBalancersInput{}, func(r *elb.DescribeLoadBalancersOutput, last bool) bool {
+		for _, lb := range r.LoadBalancerDescriptions {
+			names = append(names, *lb.LoadBalancerName)
+		}
+		return true
+	})
+	if err != nil {
+		return nil, err
```

Comment:
Would be good to wrap errors in these new "collect" functions to get some extra context.